### PR TITLE
Move System.Compiler.xml so it stops changing during every build

### PR DESCRIPTION
**Pull request by sharwell**
:tada:

### CODE REVIEW

**Comment by SergeyTeplyakov** [comment]
Awesome fix!

File: System.Compiler/System.CompilerCC.csproj [hunk 1]
```diff
@@ -228,6 +212,16 @@
...
   </PropertyGroup>
+  <PropertyGroup>
+    <!--
+      Make sure any documentation comments which are included in code get checked for syntax during the build, but do
+      not report warnings for missing comments.
+
+      CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
+    -->
+    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
+    <NoWarn>$(NoWarn),1591</NoWarn>
```
Comment (SergeyTeplyakov) [inline System.Compiler/System.CompilerCC.csproj]: I see that previous code has another "nowarn" for 1699. Do we need it as well?

Comment (sharwell) [inline System.Compiler/System.CompilerCC.csproj]: All instances of that warning were resolved sometime in the past.

Comment (SergeyTeplyakov) [inline System.Compiler/System.CompilerCC.csproj]: Understood. Merged.

**Pull request merged**